src/content/docs/en/guides/cms/keystatic.mdx

```diff
@@ -2,13 +2,263 @@
+
+:::note
+The schema of your Keystatic collections should match the collections you've defined in Astro's [content collections](/en/guides/content-collections/#defining-collections).
+
+We're working on a way to automatically sync Keystatic collections with Astro's content collections... Stay tuned! 🤗
+:::
+
+This is all Keystatic needs to start managing content, configuration-wise.
+
+Now, you need to display the Keystatic Admin UI on our site!
+
+## Setting up the Keystatic Admin UI dashboard
+
+The Keystatic Admin UI is a React app that runs in your Astro project. The following steps will set it up to live in your `/pages/` folder, and provide a dashboard for managing your content.
+
+1. Create a `keystatic.page.ts` in the project root, alongside your Keystatic config file.
+
+    ```ts
+    // keystatic.page.ts
+    import { makePage } from '@keystatic/astro/ui'
+    import keystaticConfig from './keystatic.config'
+    
+    export const Keystatic = makePage(keystaticConfig)
+    ```
+
+2. Create a new page called `src/pages/keystatic/[...params].astro` and mount the component on the client side only:
+
+    ```astro
+    ---
+    // src/pages/keystatic/[...params].astro
+    import { Keystatic } from '../../../keystatic.page'
+    export const prerender = false
+    ---
+    
+    <Keystatic client:only />
+    ```
+
+3. Create a new file called `src/pages/api/keystatic/[...params].ts` that will create `API routes` for Keystatic's Admin UI.
+
+    ```jsx
+    // src/pages/api/keystatic/[...params].ts
+    import { makeHandler } from '@keystatic/astro/api'
+    import keystaticConfig from '../../../../keystatic.config'
+    
+    export const all = makeHandler({
+      config: keystaticConfig,
+    })
+    
+    export const prerender = false
+    ```
+
+4. To launch your Keystatic Admin UI dashboard, start Astro's dev server:
+
+    ```bash
+    npm run dev
+    ```
+
+5. Visit `http://127.0.0.1:3000/keystatic` in the browser to see the Keystatic Admin UI running.
+
+## Creating a new post
+
+Try and visit the `/keystatic` page in the browser one more time, and click on the “Posts” collection.
+
+Go ahead, and create a new post.
+
+:::note
+In our Keystatic config file, you've set the `path` property for our `posts` collection to `src/content/posts/*`.
+
+As a result, creating a new post from the Keystatic Admin UI should create a new `content` directory in the `src` directory, with the new post `.mdoc` file inside.
+:::
+
+If everything worked correctly, you'll find your new post inside the `src/content/posts` directory:
+
+<FileTree title="Project Structure">
+- src/
+  - content/
+    - posts/
+      - **my-first-post.mdoc**
+</FileTree>
+
+That Markdoc file should look something like this:
+
+```markdown
+---
+title: My First Post
+---
+
+This is my very first post. I am **super** excited.
+```
+
+Niiiice ✨
+
+Time to display that post on the frontend now.
```

Comment:
``````suggestion
		<FileTree title="Project Structure">
		- src/
		  - content/
		    - posts/
		      - **my-first-post.mdoc**
		</FileTree>

5. Navigate to that file in your code editor and verify that you can see the Markdown content you entered. For example: 

		```markdown
		---
		title: My First Post
		---
		
		This is my very first post. I am **super** excited!
		```
``````